src/text/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { v, create } from '@dojo/framework/core/vdom';
+import theme from '../middleware/theme';
+
+import * as css from '../theme/default/text.m.css';
+
+export interface TextProperties {
+	size?: 'xs' | 's' | 'm' | 'l' | 'xl' | 'xxl';
+	inverse?: boolean;
+	truncated?: boolean;
+	as?: string;
+}
+
+const factory = create({ theme }).properties<TextProperties>();
+
+const Text = factory(function Text({ middleware: { theme }, properties, children }) {
+	const themedCss = theme.classes(css);
+	const { as = 'p', truncated = false, size = 'm', inverse = false } = properties();
```

Comment:
Where did the choice to default this to `p` come from? I'd have though that a `span` would be best as it's an inline text node then. Would make it useable more easily within all other widgets for consistent text styling, ie. within a `Button` / `Label` / `HelperText` etc.

Reply:
I took inspiration from the Text component in Chakra that defaulted to `p`, open for discussion :)